### PR TITLE
Update command to update /etc/host via bash.

### PR DESCRIPTION
**Pull request by ClairePhi**
Manual update is suggested if both the initial way and the way using bash do not work.

### CODE REVIEW

**Comment by jmorel** [comment]
@ClairePhi Good catch, thanks for the updated command!

Also I tested your command on zsh and it works fine there as well. You could simply replace the old command with the new one, no need to mention bash.

Could you rebase your PR?

**Comment by ClairePhi** [comment]
Thank you @jmorel! Done :)

File: docs/local_install_skaffold.md [hunk 1]
```diff
@@ -102,10 +102,10 @@ The last step will be to update `/etc/hosts` by adding this new line to it:
 192.168.65.2 substra-backend.node-1.com substra-frontend.node-1.com substra-backend.node-2.com substra-frontend.node-2.com
 ```
 
-Running the following command will do it for you (you'll be asked for your password again)
+Running the following command will do it for you (you 'll be asked for your password again):
```
Comment (jmorel) [inline docs/local_install_skaffold.md]: There's an extra space here, could you remove it before the merge?

**Comment by ClairePhi** [comment]
Extra space removed. Thanks @jmorel!

**Pull request merged**